build.gradle

```diff
@@ -799,6 +799,7 @@ project(':clients') {
     compile libs.lz4
     compile libs.snappy
     compile libs.slf4jApi
+    compile libs.jacksonDatabind // for SASL/OAUTHBEARER bearer token parsing
 
```

Comment:
we need to include public classes to javadocs at line no: 859

Reply:
👍 Fixed -- thanks for the review @omkreddy